doc/navigating.rst

```diff
@@ -38,116 +39,59 @@ so you should click it to see what it has to tell you.
-  by launching the account setup wizard (see :doc:`accountsetup`).
-* ``Modify Account`` button: Use this to change your ownCloud server settings 
-  by launching the account setup wizard (see :doc:`accountsetup`).
-
-Adding a Folder
-^^^^^^^^^^^^^^^
-
-The ``Add Folder ...`` button enables you to add a new local folder to the 
-synchronization process.
+Click **Settings** in the right-click menu to see a summary of your ownCloud 
+account settings. This shows which ownCloud account you are connected to, your 
+quota status, and a window for managing your synchronization settings.
 
-To add a new folder:
+.. image:: images/client6.png
+   :alt: Account settings window
+   
+At the top of this windows are four tabs: your account settings, which is 
+labeled with your ownCloud username, Activity, General, and Network.   
 
-1. Click the ``Add a Folder ...`` button in the Account window.
+The Activity window contains the log of your recent activities, including files 
+downloaded and deleted.
 
-  The ``Add Folder...`` window opens
+The General window has configuration options such as Launch on **System 
+Startup**, **Use Monochrome Icons**, and **Show Desktop Notifications**. This 
+is where you will find the **Edit Ignored Files** button, to launch the ignored 
+files editor, and two new features: **Ask confirmation before downloading 
+folders larger than [folder size]**, and **Add an Account**. 
 
-.. image:: images/folderwizard_local.png
+Multi-Account Support
+---------------------
 
-2. Type a *unique* path and alias name to the folder, or use the ``Choose...`` 
-   button to open a graphical file picker.
-
-.. note:: Nested synchronizations are not supported.  You
-    cannot add a folder that is already contained within another synchronized
-    folder. In addition, you cannot add a higher level (parent) folder that
-    contains a folder to which you are already synchronizing.  By default, the
-    ownCloud Set Up Wizard synchronizes your entire ownCloud account to the root
-    folder of the ownCloud server. If you wish to sync certain specific folders and not 
-    the entire root, you must first remove the root folder.
-
-3. Click 'Next' to continue.
-
-  A window opens prompting you to select a remote destination folder on the
-  ownCloud server to synchronize.
-
-.. image:: images/folderwizard_remote.png
-
-4. Select a folder on the ownCloud server to which you want to synchronize your 
-   newly added local folder.
-
-.. note:: A server folder can only be synchronized with a particular client 
-  once. If you attempt to sync the root directory, you cannot sync with 
-  other folders on the server. Similarly, if you sync with folder ``/a``, you 
-  cannot create another sync with ``/a/b``, since ``b`` is already being 
-  synched.
+You may now configure multiple ownCloud accounts in your desktop sync client. 
+Simply click the **Add an Account** button on the General tab, and follow the 
+account creation wizard.
 
```

Comment:
The new account will appear as a new tab in the settings dialog where you can adjust it's settings later.